### ISSUE TITLE
CIFAR blitz tutorial: some small improvements

### ISSUE THREAD

**Issue by murphyk** [pytorch/tutorials#534]
I have some suggestions for 
https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html

- it was initially not clear why the loop counter maxed out at 12,000. Maybe explain this is 4*12000 examples because batch size = 4, and it stops there since number of training examples is 50k. (Which can be determined from dataloader.dataset)

- Explain that the output of the network is a 4*10 tensor of floats. These are interpreted as logits by the loss function. For prediction, can convert these to probabilities using the snippet below

outputs = net(images)
probs = F.softmax(outputs, dim=1)
print(torch.sum(probs,1))

- "for i in range(4)" in the test error computation is ugly. At least rename 4 to batch_size :)

**Comment by SethHWeidman** [comment]
Hey Kevin, thanks for these great suggestions 🙂 We're working on a more comprehensive tutorial re-write and will be sure to incorporate these!
